src/main/resources/application.properties

```diff
@@ -1,3 +1,9 @@
 ## File Storage Properties
 # All files uploaded through the REST API will be stored in this directory
-file.upload-dir=src/main/resources/files/
+file.source-dir=src/main/resources/source-dir/
+file.target-dir=src/main/resources/target-dir/
+file.type-checksum=SHA-256
+file.tesseract-path=src/thridParty/Tess4J/tessdata
```

Comment:
once you update the folder name as below comment update this property

Reply:
done